error-handling/error-handlers.js

```diff
@@ -10,10 +10,17 @@ exports.customError = (err, req, res, next) => {
 
 exports.invalidError = (err,req,res,next) =>{
   if(err.code ==="22P02"|| err.code === "42703"){
+    console.log(err)
     res.status(400).send({msg: 'Bad request'})
   } else next(err)
 }
 exports.internalError = (err,req,res,next)=>{
   console.log(err)
   res.status(500).send({msg:'Internal error'})
+}
+exports.forbiddenError = (err,req,res,next)=>{
```

Comment:
As mentioned in the tests, you do not need to test for a 403, please check the comments and amend 👍